### DISCUSSION TITLE
Gotek on Greaseweazle

### DISCUSSION THREAD

**Discussion by Fluggi242** [keirf/greaseweazle#382] [General]
I run a Amiga on an RP4. Is it possible to connect the Gotek Drive over Greaseweazle on the RP4 ? Or can it put on the GPIO Port of the RP4 ?

BR Heiko

**Comment by keirf** [comment]
Yes you can connect as a PC drive to the Greaseweazle. If you use Amiberry it has built-in support for drive emulation via Greaseweazle.

**Reply by Fluggi242** [reply]
What is the right configuration ? I keep S0 + MO connectetd on the Gotek. On Amiberry Greasweazle Support is activated. Is there anything to do ?

**Reply by keirf** [reply]
Jumper S1 and JC, instead of S0 and MO. No FF.CFG required. Clear old cached config by holding both buttons with no USB stick inserted, for a few seconds.

**Reply by Fluggi242** [reply]
Thanks that fix it for me.
